### ISSUE TITLE
problem with video-simili-duplicate-cleaner, new user

### ISSUE THREAD

**Issue by ChristopherDobb** [theophanemayaud/video-simili-duplicate-cleaner#114]
After finding almost identical videos, can your app automatically mark video files with smaller bit rate? Or do I have to move them to trash one by one manually? For example: MVI_0295 (1).mov has 3727 kb/s and MVI_0295.mov has 3707 kb/s and I'd like all files with lower bit rate to be moved to album ‘Trash from Video simili duplicate cleaner’ inside Apple Photos

**Comment by theophanemayaud** [comment]
Doesn't the auto option "Move all smaller to trash" work for this case ?
![image](https://github.com/theophanemayaud/video-simili-duplicate-cleaner/assets/25605953/528ae35d-7518-4f11-a866-b6f3cf7f2211)

**Comment by ChristopherDobb** [comment]
Hi, 

Thank you for your reply.
The option you have circled below, doesn’t work. I am getting this message:

￼

**Comment by theophanemayaud** [comment]
Did you mean to include an image ?

**Comment by ChristopherDobb** [comment]
Yes, the image I included shows the message that I get when I use the option you suggested.
Instead of moving smaller to trash is moves 0 to trash

￼

**Comment by theophanemayaud** [comment]
There is no image though… 
Could you also screenshot the properties of an example pair of videos that you would want it to auto trash so I can see what’s up ?

**Comment by ChristopherDobb** [comment]
Hi Theopane, 

I just started using your app and I ran into some problems.
I scanned my Photos library on mac using the top option in auto settings to find duplicates.
It took over 12 hours and your app found duplicate 1341 videos. They were moved to "Trash from video simili" folder
I deleted all files using select all, command+delete. Then I deleted everything from “Recently Deleted” folder in Photos.

I ran your app again to scan the same library, with the same settings, and I expected that no more duplicates will be found because the same scan was just completed. However, the app shows that it found a lot more duplicates and is going again through a very slow process.

Do you have to run your Photos library scan several times to find all duplicates? I did not change the settings from the first scan.

Thank you very much for your help.

Christopher 

Here are the screenshots:

￼

￼

**Comment by theophanemayaud** [comment]
Hi !! I again didn’t receive the screenshots 😬

Anyways : is your Photo Library connected to icloud ? If so, in the photos settings, did you enable the option to download all locally ? Otherwise you’re only comparing downloaded elements, which might change depending on what you accessed recently, and storage. 

Also, because of the somewhat random nature of video decoding, a few more duplicates might be found during a second run. But it shouldn’t be the majority !

**Comment by ChristopherDobb** [comment]
Hi Theophane, 

Thank you for your reply.
How do I attach the screenshots so that you can see them?
I sent a copy to my work email and I can see them there.

Photo Library I am working with is not connected to iCloud.
You app is still comparing photos after 18 hours, so it’s not just a few more duplicates but thousands.

Christopher

**Comment by theophanemayaud** [comment]
For the photos I guess you have to come back to GitHub and write here directly (I’m not replying from mail myself so I don’t know)

As for the scan, what stage is it at ? Because there are two stages : 

1. The app finds all files then extracts reference images. This process is the most intensive and time consuming, as it must process every video. 

2. Only then can it actually compare videos and look for duplicated. This second step is started only when the popup window opens with the two videos side by side (the app might also freeze for some time when opening that popup window : it is looking for the first pair of duplicates)

So at what step are you taking 18 hours ? If it is the first, it’s normal even if there are no duplicates. It must rescan each time (do you have cache enabled ? Cache should make it much much faster, but still take about 25% of the original time)

**Comment by ChristopherDobb** [comment]
The first stage took about 15-20 min and when it finished it showed this:

11491 intact videos out of 11990 total
Found 2519 videos (118.5 GB) with one or more matches.

Since yesterday it has been on the second step where I can see 2 videos side by side, slowly moving through all of them.

When I scanned the library the first time, day before yesterday, it removed about 1,340 videos to Trash Folder, then I deleted it correctly, then deleted from Recently Deleted in Photos Library.

**Comment by ChristopherDobb** [comment]
Hi again.
There is another problem with Video Simili app.
It froze on a large library so I force quit.
I tried it on a smaller 150 GB library and after if finds duplicates, it doesn’t allow me to select manual or auto, and therefor it doesn’t allow me to select on of the 3 options in the settings.
It just keep showing spinning wheel.
When I go to check force quit option, it says Video Simili not responding.
I restarted the computer and the same behavior repeats.
